solutions/usaco-831.mdx

```diff
@@ -2,13 +2,125 @@
 id: usaco-831
 source: USACO Bronze 2018 Open
 title: Team Tic Tac Toe
-author: Jesse Choe, Kevin Sheng, Benjamin Qi
+author: Jesse Choe, Kevin Sheng, Benjamin Qi, Brad Ma
 ---
 
 ## Solution 1
 
 [Official Analysis](http://www.usaco.org/current/data/sol_tttt_bronze_open18.html)
 
+<LanguageSection>
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class TeamTicTacToe {
+	static char[][] Board = new char[3][3];
```

Comment:
wth
why did you call it `Board` and not `board`

Reply:
^^^

Reply:
what's wrong with `Board`?

Reply:
syntax coloring

Reply:
its java convention for variables to be lower camel case, also Board screws up the coloring because it thinks that it's a class